modules/installation-cloudformation-subnet-localzone.adoc

```diff
@@ -14,53 +14,6 @@ you need for your {product-title} cluster that uses AWS Local Zones.
 ====
 [source,yaml]
 ----
-# CloudFormation template used to create Local Zone subnets and dependencies
-AWSTemplateFormatVersion: 2010-09-09
-Description: Template for create Public Local Zone subnets
-
-Parameters:
-  VpcId:
-    Description: VPC Id
-    Type: String
-  ZoneName:
-    Description: Local Zone Name (Example us-east-1-nyc-1a)
-    Type: String
-  SubnetName:
-    Description: Local Zone Name (Example cluster-public-us-east-1-nyc-1a)
-    Type: String
-  PublicRouteTableId:
-    Description: Public Route Table ID to associate the Local Zone subnet
-    Type: String
-  PublicSubnetCidr:
-    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-4]))$
-    ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-24.
-    Default: 10.0.128.0/20
-    Description: CIDR block for Public Subnet
-    Type: String
-
-Resources:
-  PublicSubnet:
-    Type: "AWS::EC2::Subnet"
-    Properties:
-      VpcId: !Ref VpcId
-      CidrBlock: !Ref PublicSubnetCidr
-      AvailabilityZone: !Ref ZoneName
-      Tags:
-      - Key: Name
-        Value: !Ref SubnetName
-      - Key: kubernetes.io/cluster/unmanaged
-        Value: "true"
-
-  PublicSubnetRouteTableAssociation:
-    Type: "AWS::EC2::SubnetRouteTableAssociation"
-    Properties:
-      SubnetId: !Ref PublicSubnet
-      RouteTableId: !Ref PublicRouteTableId
-
-Outputs:
-  PublicSubnetIds:
-    Description: Subnet IDs of the public subnets.
-    Value:
-      !Join ["", [!Ref PublicSubnet]]
+include::https://raw.githubusercontent.com/openshift/installer/master/upi/aws/cloudformation/01.99_net_local-zone.yaml[]
```

Comment:
@mtulio . OK to link to the snippet on another repo/server provided that the source is public facing as is not behind some sort of privacy setting.

Reply:
Here is what a colleague stated:

> As long as it's part of their QE testing, since it's from a repo in the openshift org, it'll be ok. Make sure that you're linking to the most appropriate one version-wise.

Reply:
^^ Can you confirm that your snippet meets these requirements?

Reply:
Thanks for confirming, @dfitzmau . 

This is pointing to installer where there are jobs (presubmits) consuming/testing it.

> you're linking to the most appropriate one version-wise.

I think we need to pin to the correct version, 4.14, but I would like to know if docs have some procedure to update those lines for each release. If we point to the 4.14 branch in the installer, the docs in 4.15 will keep pointing to 4.14? Can we use some dynamic variable instead of a pinned version?

cc @yunjiang29 for QE testing

Reply:
Hi @mtulio . Our only guidelines are https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#embedding-an-external-file . 

Notice the example link in ^^ (include::https://raw.githubusercontent.com/openshift/installer/release-4.8/upi/azure/01_vnet.json[]) has the OCP version in the URL. If the file was present for each targeted release branch on the host repository, this would solve the 4.14 and 4.15 need.

Reply:
> Can we use some dynamic variable instead of a pinned version?

Same question, the ideal way is to specify versioned-template.